src/js/components/FileInput/FileInput.js

```diff
@@ -328,6 +330,7 @@ const FileInput = forwardRef(
                   icon={<RemoveIcon />}
                   hoverIndicator
                   onClick={(event) => {
+                    if (onClickRemove) if (!onClickRemove()) return;
```

Comment:
I think we should also enable working with other forms of confirmation, such as opening a Layer. In that case, perhaps the call signature could be something like: `onRemove(file, () => { ... do the remove stuff ... })`

Reply:
@DouglasPds will this functionality be possible with your current implementation? Or will updates to the code be needed